recipes/libpcsclite/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "libpcsclite" %}
+{% set version = "1.9.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://salsa.debian.org/rousseau/PCSC/-/archive/{{ version }}/PCSC-{{ version }}.tar.gz
+  sha256: 856a1daed9a5ae1ac265c75ce9f81743e66a972e5be5a1d3a2196ed23f75ea26
+
+build:
+  skip: True  # [not linux]
+  script:
+    - cd PCSC
+    - ./bootstrap
+    - ./configure
+    - make
+  number: 0
+
+requirements:
+  build:
+    - autoconf-archive
+    - flex
+    - libsystemd
+    - libusb
+    - make
+  host:
+  run:
+
+test:
+  commands:
+    # Placeholder
+    - ls
+
+about:
+  home: https://pcsclite.apdu.fr/
+  dev_url: https://pcsclite.apdu.fr/
+  summary: Middleware to access a smart card using SCard API (PC/SC).
+  license: BSD-3-Clause and GPL-3.0-or-later and BSD-2-Clause and MIT
+  license_file: COPYING
```

```suggestion
  license_file: 
    - GPL-3.0.txt
    - COPYING
```
There are some more here that probably need to be linked: https://github.com/LudovicRousseau/PCSC/blob/master/COPYING

---

The text of all the linked license is in COPYING though, right?

---

Ahh true yes, read to quickly. Just including Copying and updating the license field is enough.